synapse/models/economic.py

```diff
@@ -134,6 +140,20 @@ def getModelDefs(self):
                         'doc': 'The econ:price of the purchase'}),
                 )),
 
+                ('econ:receipt:item', {}, (
+
+                    ('purchase', ('econ:purchase', {}), {
+                        'doc': 'The purchase that contains this line item.'}),
```

Comment:
maybe "the purchase of this line item"?

Reply:
"The purchase that contains the purchase of this line item?" seems a bit redundant. Did i misinterpret?